repos/system_upgrade/el7toel8/actors/targetuserspacecreator/libraries/userspacegen.py

```diff
@@ -58,40 +59,41 @@ def _get_product_certificate_path():
     """
     Retrieves the required / used product certificate for RHSM.
     """
-    variant = api.current_actor().configuration.os_release.variant_id
     architecture = api.current_actor().configuration.architecture
     target_version = api.current_actor().configuration.version.target
-    # TODO: so far only base channel is available in rhel8
-    target_channel = 'base'
-    certs_folder = api.get_common_folder_path(PROD_CERTS_FOLDER)
+    target_product_type = get_product_type('target')
+    certs_dir = api.get_common_dir_path(PROD_CERTS_FOLDER)
 
+    # TODO: do we need EUS/... here or is it ga one enough to get eus repos?
```

Comment:
The EUS prod certs have different numbers: https://docs.google.com/document/d/1UQ3a0GG7kN3_-isb6Bd612UX4BgtLRS0sWPkeMMQ6ds/edit#heading=h.z4wi73tiq5hw
I think you can remove the TODO.

Reply:
I know, but the TODO is about: are we able to get eus content using a ga certificate or do we have to use eus certificates?.. as well, missing the detection whether we are on eus or not know. so keeping the todo until we know answers.